tools/vms-generator/utils/utils.go

```diff
@@ -83,9 +83,10 @@ const (
 	imageCirros = "cirros-container-disk-demo"
 	imageFedora = "fedora-cloud-container-disk-demo"
 )
-
 const windowsFirmware = "5d307ca9-b3ef-428c-8861-06e72d69f223"
 const defaultInterfaceName = "default"
+const enableNetworkInterfaceMultiqueueForTemplate = true
+const CPUModelToEnableMultiqueue = "virtio"
```

Comment:
it's not CPU model, it's ethernet adaptor model. (e1000 is an Ethernet driver; same for virtio - in this context.)

Reply:
Thanks for the clarification. Done